stable/xwiki/values.yaml

```diff
@@ -0,0 +1,47 @@
+# app specific
+
+db:
+  user: xwiki
+  pass: xwiki
+  host: localhost
+  database: xwiki
+
+persistence:
+  enabled: false
+  size: 1GiB
```

Comment:
This should be 1G

Reply:
https://is.gd/JtAEMn says I am wrong but I can still use "Gi" :) . I'd like to use the power-of-two values, cause it's more binary.

Reply:
https://is.gd/JtAEMn says I am wrong but apparently I can still use "Gi" :) . I'd like to use the power-of-two values, cause it's more binary.